examples/connectivity/plot_pac.py

```diff
@@ -0,0 +1,195 @@
+Simulate phase-amplitude coupling between two signals, and computes
+several PAC metrics between them. Calculates PAC for all timepoints, as well
+as for time-locked PAC responses.
+References
+----------
+[1] Canolty RT, Edwards E, Dalal SS, Soltani M, Nagarajan SS, Kirsch HE,
+    Berger MS, Barbaro NM, Knight RT. "High gamma power is phase-locked to
+    theta oscillations in human neocortex." Science. 2006.
+[2] Tort ABL, Komorowski R, Eichenbaum H, Kopell N. Measuring phase-amplitude
+    coupling between neuronal oscillations of different frequencies. Journal of
+    Neurophysiology. 2010.
+"""
+# Author: Chris Holdgraf <choldgraf@berkeley.edu>
+#
+# License: BSD (3-clause)
+import mne
+import numpy as np
+from matplotlib import pyplot as plt
+from mne_sandbox.connectivity import (simulate_pac_signal,
+                                      phase_amplitude_coupling)
+
+print(__doc__)
+np.random.seed(1337)
+
+###############################################################################
+# Phase-amplitude coupling (PAC) is a technique to determine if the
+# amplitude of a high-frequency signal is locked to the phase
+# of a low-frequency signal. The phase_amplitude_coupling function
+# calculates PAC between pairs of signals for one or multiple
+# time windows. In this example, we'll simulate two signals. One
+# of the signals has an amplitude that is locked to the phase of
+# the other. We'll calculate PAC for a number of time points, and
+# in both directions to show how PAC responds.
+
+# Define parameters for our simulated signal
+sfreq = 1000.
+f_phase = 5
+f_amp = 40
+frac_pac = .99  # This is the fraction of PAC to use
+mag_ph = 4
+mag_am = 1
+
+# These are the times where PAC is active in our simulated signal
+n_secs = 20.
+time = np.arange(0, n_secs, 1. / sfreq)
+event_times = np.arange(1, 18, 4)
+event_dur = 2.
+
+# Create a time mask that defines when PAC is active
+msk_pac_times = np.zeros_like(time).astype(bool)
+for i_time in event_times:
+    msk_pac_times += mne.utils._time_mask(time, i_time, i_time + event_dur)
+
+# Now simulate two signals. First, a low-frequency phase
+# that modulates high-frequency amplitude
+_, lo_pac, hi_pac = simulate_pac_signal(time, f_phase, f_amp, mag_ph, mag_am,
+                                        frac_pac=frac_pac,
+                                        mask_pac_times=msk_pac_times)
+
+# Now two signals with no relationship between them
+_, lo_none, hi_none = simulate_pac_signal(time, f_phase, f_amp, mag_ph,
+                                          mag_am, frac_pac=0,
+                                          mask_pac_times=msk_pac_times)
+
+# Finally we'll mix them up.
+# The low-frequency phase of signal A...
+signal_a = lo_pac + hi_none
+# Modulates the high-frequency amplitude of signal B. But not the reverse.
+signal_b = lo_none + hi_pac
+
+
+# To standardize the scales of each PAC metric
+def normalize_pac(pac):
+    return (pac - np.min(pac)) / (np.max(pac) - np.min(pac))
```

Comment:
I would not subtract the min, as a metric is supposed to be zero when there is no coupling.
In other words, the zero value has a meaning.
I would just divide by the max (or the mean), so as to solve the scaling difference between metrics.